Deployment/Milyli.ScriptRunner.Services.Interfaces.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Milyli.ScriptRunner.Services.Interfaces</id>
+	<version>1.0.0.0</version>
+    <title>ScriptRunner Services</title>
+    <authors>Milyli inc</authors>
+    <projectUrl>https://github.com/Milyli/RelativityScriptRunner</projectUrl>
+    <owners>Milyli inc</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Kepler services to interact with Milyli ScriptRunner</description>
+  </metadata>
+  <files>
+    <file src=".\Artifacts\Milyli.ScriptRunner.Services.Interfaces.dll"
+	target=".\Artifacts\Libs\Milyli.ScriptRunner.Services.Interfaces.dll" />
```

Comment:
These targets don't seem to be correct [according to conventions](https://docs.microsoft.com/en-us/nuget/reference/nuspec#specifying-files-to-include-in-the-package)

> The relative path to the folder within the package where the source files are placed, which must begin with lib, content, build, or tools. See Creating a .nuspec from a convention-based working directory.

  - [ ] Fix the targets